packages/paste-website/src/pages/getting-started/engineering/index.mdx

```diff
@@ -336,6 +273,96 @@ This is a free font that is [hosted on Google Fonts](https://fonts.google.com/sp
 [how we load the font](https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/assets/css/fonts.css#L105)
 into your own app if you are using the default theme.
 
+<Box marginBottom="space140" />
+
+## Additional information
+
+### Core package list
+
+Core gives you the following packages. You can install these packages individually as well.
+
+<Disclosure>
+  <DisclosureHeading as="h4" variant="heading50">
+    Show full package list
+  </DisclosureHeading>
+  <DisclosureContent>
+    <Table tableLayout="fixed">
+      <THead>
+        <Tr>
+          <Th>Package</Th>
+          <Th>Version</Th>
+        </Tr>
+      </THead>
+      <TBody>
+        {Object.keys(coreDependencies).map((dependency, index) => {
+          return (
+            <Tr key={`core-dep-${dependency}-${index}`}>
+              <Td>{dependency}</Td>
+              <Td>{coreDependencies[dependency]}</Td>
+            </Tr>
+          );
+        })}
+      </TBody>
+    </Table>
+  </DisclosureContent>
+</Disclosure>
+
+<Box marginBottom="space100" />
+
+### How the Core package stays lean
```

Comment:
I'm not sure if this belongs on the getting started page. How do we feel about this living somewhere a little more dedicated to core? (I have no idea where that is)

Reply:
I was struggling with where to put it honestly. If we had a blog post we could reference then I'd link to that. Kinda why I tossed it way at the bottom here. Open to suggestions.

Reply:
For now it's fine, but it should be pretty easy for us to just add a blog section. We could then host @aayushpi's post on the site too.